cypher/cypher-docs/src/docs/dev/ql/administration/security/deny-read-syntax.asciidoc

```diff
@@ -1,13 +0,0 @@
-.Command syntax
```

Comment:
Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/privileges-read.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/privileges-reads/#access-control-privileges-reads-read